--- akregator-17.08.0/src/akregator_part.cpp.0000~	2017-08-22 12:50:19.337160627 +0200
+++ akregator-17.08.0/src/akregator_part.cpp	2017-08-22 13:46:35.738442762 +0200
@@ -131,6 +131,16 @@ static QDomDocument createDefaultFeedLis
     apps.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("https://store.kde.org/content.rdf"));
     mainFolder.appendChild(apps);
 
+    // OpenMandrivaAssociation blog feeds
+    QDomElement omaFolder = doc.createElement(QStringLiteral("outline"));
+    omaFolder.setAttribute(QStringLiteral("text"), QStringLiteral("OpenMandriva Association"));
+    body.appendChild(omaFolder);
+
+    QDomElement omaNews = doc.createElement(QStringLiteral("outline"));
+    omaNews.setAttribute(QStringLiteral("text"), i18n("Blog and News"));
+    omaNews.setAttribute(QStringLiteral("xmlUrl"), QStringLiteral("https://www.openmandriva.org/spip.php?page=backend"));
+    omaFolder.appendChild(omaNews);
+
     // Brazilian Portuguese feeds
     QDomElement portugueuseFolder = doc.createElement(QStringLiteral("outline"));
     portugueuseFolder.setAttribute(QStringLiteral("text"), i18n("Brazilian Portuguese feeds"));
